ophyd/status.py

```diff
@@ -40,14 +46,44 @@ class StatusBase:
         The amount of time to wait between the caller specifying that the
         status has completed to running callbacks
     """
-    def __init__(self, *, timeout=None, settle_time=None, done=False,
+    # Theory of operation:
```

Comment:
Fair. I was inspired by trio which had full-on esoteric essays in the module and class comments. I worry that this level of detail might scare away a user would might wonder if this will be on the test. I will give it some thought.

Reply:
K, I'm convinced.